package.json

```diff
@@ -9,13 +9,22 @@
   ],
   "scripts": {
     "test": "echo \"Error: no test specified\" && exit 1",
-    "npm-publish": "npm publish --access public"
+    "npm-publish": "npm publish --access public",
+    "start": "parcel serve examples/index.html"
   },
   "author": "",
   "repository": "git://github.com/mercurymedia/elm-ag-grid.git",
   "license": "MIT",
   "peerDependencies": {
     "ag-grid-community": "^26.2.1",
     "ag-grid-enterprise": "^26.2.1"
+  },
+  "devDependencies": {
+    "@parcel/transformer-elm": "^2.3.2",
+    "@webcomponents/custom-elements": "^1.5.0",
+    "ag-grid-community": "^26.2.1",
+    "ag-grid-enterprise": "^26.2.1",
```

Comment:
Are these dependencies for the example app? I thought we don't want to have them as dependencies in here...

Reply:
These are just devDependencies. So they will not be required when using the package. It's just for local development and running the example app.